.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+name: Draft Release
+
+on:
+  push:
+    tags: '[0-9]+.[0-9]?[0-9]?[0-9]?.?[0-9]+'
```

Comment:
```suggestion
    // https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
    tags: '^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'
```

Reply:
Also maybe adjust if we need a `v` prefix.

Reply:
Unfortunately, these are not regexes, more like slightly extended glob patterns. Check the syntax cheat sheet at:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Note that I explicitly want to prevent that any non-full releases get created. Only full version releases (which do not start with a v or have any suffixes) should trigger this process. The goal is to save me work, not create extra work for me to clean up drafts.

Here is a full list of all the tags I wanted to support with this pattern, for reference:
```
$ git tag
0.18
0.19
0.20
0.21
0.21.1
0.22
1.0
1.1
1.1.1
1.2
1.2.1
1.2.2
1.2.3
1.3
1.3.1
1.3.2
1.3.3
1.3.4
1.3.5
1.4.0
1.5.0
1.5.1
1.5.2
1.6.0
```

Reply:
Hmm but pre-releases would be fine to trigger it, too IMHO. I mean after all, when do we do that? Likely never. (as you have shown we never did in the past)

But yeah, this was just an idea to be able to test it without triggering a real release. It is a draft release anyway so…

Reply:
We could do a patch release in a few days with the RTL changes - having this workflow will reduce my work doing the releases and so as I had outlined, if it is less work, I don't mind releasing more frequently. I mind having to spend half a day on it.